runs/update_local_display.sh

```diff
@@ -37,7 +37,7 @@ else
 fi
 
 if rotation=$(mosquitto_sub -p 1886 -t "openWB/optional/int_display/rotation" -C 1 -W 1); then
-	rotationValue=$(((rotation / 90 + 4) % 4))  # this allows negative rotation angles
+	rotationValue=$(((rotation / 90 + 4) % 4)) # this allows negative rotation angles
```

Comment:
Default für die Rotation ist 180° (Standalone) Würde bei allen erkannten Series-Display das Display auf dem Kopf stehen?

Reply:
Gerade nochmal nachgesehen:
- series2: 0°
- standalone: 180°
Ist also die Frage, was man als Standard nehmen will. Für einen Gerätetyp muss die Einstellung geändert werden. Das ist aber mit diesem PR nicht neu/anders als vorher.

Reply:
Ich denke, dann verlagert sich das Problem von "Das Display ist kaputt, weil es zeigt nichts an" zu "Die Box ist falsch zusammengebaut, das Display steht auf dem Kopf, ich schicks zurück". Kann man nicht prüfen, ob ttySUB0 oder ttyACM0 vorhanden ist und dann die Rotation auf 0° setzen und sonst auf 180°?

Reply:
Ich nehme jetzt 0° als neuen Standardwert. Somit sollte der Großteil unserer Produkte direkt richtig konfiguriert sein. Lediglich eine Standalone benötigt dann manuelle Konfiguration. Das kann ggf. in der Produktion vorkonfiguriert werden.